pom.xml

```diff
@@ -56,7 +56,7 @@
     <surefire.redirectTestOutputToFile>true</surefire.redirectTestOutputToFile>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!-- version properties -->
-    <cdap.version>6.8.0</cdap.version>
+    <cdap.version>6.8.0-SNAPSHOT</cdap.version>
```

Comment:
why are you depending on SNAPSHOT version?

Reply:
change to 6.8.0

Reply:
Done. Changed it to 6.8.0